definitions/npm/react-apollo_v2.x.x/flow_v0.58.x-/react-apollo_v2.x.x.js

```diff
@@ -276,16 +276,16 @@ declare module 'react-apollo' {
 
   declare export class Subscription<TData> extends React$Component<SubscriptionProps<TData>> {}
 
-  declare export type MutationFunction<TVariables> = (options: {
+  declare export type MutationFunction<TData=void, TVariables=void> = (options: {
```

Comment:
The react-apollo Typescript has these defaulted to `TData=any` and `TVariables=OperationVariables` where OperationVariables is `{[key: string]: any}`

https://github.com/apollographql/react-apollo/blob/master/src/Mutation.tsx#L50

Any reason not to follow that?